plugins/rapid7_intsights/unit_test/util.py

```diff
@@ -30,6 +30,10 @@ def read_file_to_string(filename):
         with open(filename) as my_file:
             return my_file.read()
 
+    @staticmethod
+    def read_file_to_dict(filename):
+        return json.loads(Util.read_file_to_string(os.path.join(os.path.dirname(os.path.realpath(__file__)), filename)))
+
     @staticmethod
     def mock_request(*args, **kwargs):
         class MockResponse:
```

Comment:
Is this `class MockResponse` indented too far?

Reply:
this is allowed- its like an "inner class" type pattern seen in other languages / use cases. It means only this function uses this class. It also means that if there were any other MockResponse class found in an import in this file, this locally defined MockResponse would still be used instead of that imported one. Its a way of ensuring this function will always use this exact MockResponse class

Reply:
Similar to this, you can also define functions within functions @jrose-r7 !